CMakeLists.txt

```diff
@@ -91,6 +94,8 @@ if (CMAKE_COMPILER_IS_GNUCXX)
 	endif ()
 endif ()
 
+set(CMAKE_REQUIRED_FLAGS ${TEMP_REQUIRED_FLAGS} )
```

Comment:
This seems superfluous, it's overwritten with the next command.

Reply:
Removed it. Thanks.